npm/src/registry.test.ts

```diff
@@ -10,6 +10,10 @@ describe('Registry', () => {
     const usdcId = base64ToUint8Array('reum7wQmk/owgvGMWMZn/6RFPV24zIKq3W6In/WwZgg=');
     const res = registry.getMetadata(new AssetId({ inner: usdcId }));
     expect(res.base).toEqual('wtest_usd');
+    expect(res.badges.length).toEqual(1);
+    expect(res.badges[0]?.png).toEqual(
+      'https://raw.githubusercontent.com/prax-wallet/registry/main/images/penumbra-favicon.png',
+    );
```

Comment:
test coverage can be expanded – assets without badges or missing metadata.

Reply:
Missing metadata test already there. Added test for getting asset without badge.